httpbin/helpers.go

```diff
@@ -166,6 +166,24 @@ func parseBoundedDuration(input string, min, max time.Duration) (time.Duration,
 	return d, err
 }
 
+// Returns a new rand.Rand from the given seed string.
+func parseSeed(rawSeed string) (*rand.Rand, error) {
+	var seed int64
+	if rawSeed != "" {
+		var err error
+		seed, err = strconv.ParseInt(rawSeed, 10, 64)
+		if err != nil {
+			return nil, err
+		}
+	} else {
+		seed = time.Now().Unix()
```

Comment:
Heh, in testing this locally I realized there's a (preexisting) issue with using `Unix()` here:  It has one second granularity, which leads to unexpectedly periodic behavior in this endpoint by default:

```
$ while true; do curl -v http://localhost:8080/unstable 2>&1 | grep '< HTTP'; sleep 0.2; done
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 200 OK
< HTTP/1.1 200 OK
< HTTP/1.1 200 OK
< HTTP/1.1 200 OK
< HTTP/1.1 200 OK
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 200 OK
< HTTP/1.1 200 OK
< HTTP/1.1 200 OK
< HTTP/1.1 200 OK
< HTTP/1.1 200 OK
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
^C
```

Changing this to `UnixNano()` leads to a better distribution of results:

```
$ while true; do curl -v http://localhost:8080/unstable 2>&1 | grep '< HTTP'; sleep 0.2; done
< HTTP/1.1 200 OK
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 200 OK
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 200 OK
< HTTP/1.1 200 OK
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 200 OK
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 200 OK
< HTTP/1.1 200 OK
< HTTP/1.1 200 OK
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 200 OK
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 200 OK
< HTTP/1.1 200 OK
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
```